docs/manual_doc/Statement_Syntax/Data_Administration_Statements/Configuration_Statements/gflag_Syntax.md

```diff
@@ -0,0 +1,58 @@
+Nebula use `gflags` for run-time configurations.
+
+
+### Show variables
+```
+SHOW VARIABLES [graph|meta|storage]
+```
+
+For example
+
+```
+(user@127.0.0.1) [(none)]> SHOW VARIABLES meta
+============================================================================================================================
+| module | name                                        | type   | mode      | value                                        |
+============================================================================================================================
+| META   | v                                           | INT64  | IMMUTABLE | 4                                            |
+----------------------------------------------------------------------------------------------------------------------------
+| META   | help                                        | BOOL   | IMMUTABLE | False                                        |
+----------------------------------------------------------------------------------------------------------------------------
+| META   | port                                        | INT64  | IMMUTABLE | 45500                                        |
+----------------------------------------------------------------------------------------------------------------------------
+```
+
+
+### Get variables
+```
+GET VARIABLES [graph|meta|storage : var]
+```
+
+For example
+```
+(user@127.0.0.1) [(none)]> GET VARIABLES storage:load_config_interval_secs
+=================================================================
+| module  | name                      | type  | mode    | value |
+=================================================================
+| STORAGE | load_config_interval_secs | INT64 | MUTABLE | 120   |
+-----------------------------------------------------------------
+```
+
+### Update variables
+```
+UPDATE VARIABLES [graph|meta|storage : var = value]
```

Comment:
If `UPDATE VARIABLES var = value`, we can set all module of var to value (if it is mutable). Also we can `GET VARIABLES var` to show `var` in all module.